src/Compilers/CSharp/Test/Emit2/Semantics/ExperimentalAttributeTests.cs

```diff
@@ -2076,9 +2074,9 @@ void M(C c)
         if (inSource)
         {
             comp.VerifyDiagnostics(
-                // 0.cs(3,12): warning DiagID1: 'C' is for evaluation purposes only and is subject to change or removal in future updates.
+                // 0.cs(3,12): error DiagID1: 'C' is for evaluation purposes only and is subject to change or removal in future updates.
                 //     void M(C c)
-                Diagnostic("DiagID1", "C").WithArguments("C").WithLocation(3, 12)
+                Diagnostic("DiagID1", "C").WithArguments("C").WithLocation(3, 12).WithWarningAsError(true)
```

Comment:
Do we have a test verifying that .editorconfig can suppress the diagnostic?

Reply:
Added tests for editorconfig and nowarn. That caught a bug. Thanks!